order/models.py

```diff
@@ -0,0 +1,41 @@
+from tkinter import CASCADE
+from django.db import models
+from core import TimeStampModel
+
+# Create your models here.
+class Cart(models.Model):
+    products_count    = models.IntegerField(max_length=50)
```

Comment:
수량은 quantity 로 바꾸는 것이 의미가 더 정확할 것 같습니다.
IntegerField에 max_length 옵션은 없어도 됩니다. 제거해 주세요.
```suggestion
    quantity    = models.IntegerField()
```